src/screens/BusesScreen.tsx

```diff
@@ -0,0 +1,150 @@
+import { useEffect } from 'react';
+import { useSelector } from 'react-redux';
+import AuthLayout from 'layouts/AuthLayout';
+
+export const data = [
+  {
+    id: 1,
+    name: 'Jeremy Guillon',
+    email: 'Admin@safebear.com',
+    role: 'Admin',
+  },
+  {
+    id: 2,
+    name: 'Jane Doe',
+    email: 'Admin@safebear.com',
+    role: 'Editeur',
+  },
+  {
+    id: 3,
+    name: 'Jacob Marcus',
+    email: 'Admin@safebear.com',
+    role: 'Admin',
+  },
+  {
+    id: 4,
+    name: 'John Doe',
+    email: 'Admin@safebear.com',
+    role: 'Modérateur',
+  },
+  {
+    id: 5,
+    name: 'Marius sol',
+    email: 'Admin@safebear.com',
+    role: 'Editeur',
+  },
+  {
+    id: 6,
+    name: 'Jeremy Guillon',
+    email: 'Admin@safebear.com',
+    role: 'Modérateur',
+  },
+];
+export interface IAdmin {
+  id: number;
+  name: string;
+  email: string;
+  role: string;
+}
+
+export default function Buses() {
+  const pageSize = 7;
+  const linkCount = 3;
+  const users = data;
+  const search = '';
+  const pageData: IAdmin[] = [];
+  const currentPage = 1;
+  const currentLinkPage = 1;
+  let selected: number[] = [];
```

Comment:
this should have been put into a useState. This is react bruh

Reply:
Yh, that is sth I will do in my next push